mpdfunk.py

```diff
@@ -0,0 +1,59 @@
+"""python-mpd2 library"""
```

Comment:
![image](https://user-images.githubusercontent.com/70344584/233822238-f9e84ae1-7e0c-4d1e-9680-23b0f0d4589f.png)

We should describe what mpdfunk purpose is here